transmogrifier/helpers.py

```diff
@@ -1,34 +1,33 @@
 import logging
 from datetime import UTC, datetime
 
+import transmogrifier.models as timdex
 from transmogrifier.config import DATE_FORMATS
 
 logger = logging.getLogger(__name__)
 
 
-def generate_citation(extracted_data: dict) -> str:
+def generate_citation(timdex_record: timdex.TimdexRecord) -> str:
```

Comment:
Nice changes related to this passing of instantiated `timdex_record` vs `extracted_data` dict. 😎